react/features/breakout-rooms/components/native/CollapsibleList.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import React, { useCallback, useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import { Icon, IconArrowDown, IconArrowUp } from '../../../base/icons';
+import { StyleType } from '../../../base/styles';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The children to be displayed within this list.
+     */
+    children: React$Node,
+
+    /**
+     * Additional style to be appended to the CollapsibleList container.
+     */
+    containerStyle?: StyleType,
+
+    /**
+     * Callback to invoke when the {@code CollapsibleList} is long pressed.
+     */
+    onLongPress?: Function,
+
+    /**
+     * Collapsable list title.
+     */
+    title: Object
+}
+
+export const CollapsibleList
+    = ({ children, containerStyle, onLongPress, title }: Props) => {
+        const [ collapsed, setCollapsed ] = useState(true);
+        const _toggleCollapsed = useCallback(() => {
+            setCollapsed(!collapsed);
+        }, [ collapsed ]);
+
+        return (
+            <View style = { !collapsed && containerStyle }>
```

Comment:
Isn't it a bit weird to not use the style when collapsed?

Reply:
The style is applied when the list is collapsed, but I need to change a bit the things here.